Newest version doesn't build anymore

Today I am getting the following error when running make.
```
[  2%] Building CXX object CMakeFiles/yaml-cpp.dir/src/convert.cpp.o
In file included from /home/git/actions-runner/_work/dep/yaml-cpp/src/convert.cpp:3:
/home/git/actions-runner/_work/dep/yaml-cpp/include/yaml-cpp/node/convert.h:20:10: fatal error: version: No such file or directory
 #include <version>
          ^~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/yaml-cpp.dir/build.make:102: CMakeFiles/yaml-cpp.dir/src/convert.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:912: CMakeFiles/yaml-cpp.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```
I believe this is due to commit 6262201182a0ee357d8a456e6d1a8e5984e4c6dd

--- a/include/yaml-cpp/node/convert.h
+++ b/include/yaml-cpp/node/convert.h
@@ -17,9 +17,8 @@
 #include <type_traits>
 #include <valarray>
 #include <vector>
-#include <version>
 
-#ifdef __cpp_lib_string_view
+#if __cplusplus >= 201703L
 #include <string_view>
 #endif
 
@@ -94,7 +93,7 @@ struct convert<char[N]> {
   static Node encode(const char* rhs) { return Node(rhs); }
 };
 
-#ifdef __cpp_lib_string_view
+#if __cplusplus >= 201703L
 template <>
 struct convert<std::string_view> {
   static Node encode(std::string_view rhs) { return Node(std::string(rhs)); }

